openff/toolkit/utils/toolkit_registry.py

```diff
@@ -162,24 +163,19 @@ def register_toolkit(self, toolkit_wrapper, exception_if_unavailable=True):
         if isinstance(toolkit_wrapper, type):
             try:
                 toolkit_wrapper = toolkit_wrapper()
+
+            # This exception can only be raised by OpenEyeToolkitWrapper
+            except LicenseError as openeye_exception:
+                if exception_if_unavailable:
+                    raise ToolkitUnavailableException(openeye_exception)
+                else:
+                    logger.warning(openeye_exception)
```

Comment:
```suggestion
            # This exception can be raised by OpenEyeToolkitWrapper
            except LicenseError as license_exception:
                if exception_if_unavailable:
                    raise ToolkitUnavailableException(license_exception.msg)
                else:
                    logger.warning(license_exception)
```

I'm a little wary of the possibility that another proprietary toolkit could be added in the future (though this seems unlikely and nitpicky). 

BLOCKING: More importantly, `LicenseError` objects are not strings and so can't be passed to `ToolkitUnavailableException` - it gives a nasty error.

An alternative might be to make `LicenseError` a subclass of `ToolkitUnavailableException`, and simply re-raise it.